packages/odyssey-react-mui/src/labs/DatePickers/DatePicker.tsx

```diff
@@ -308,13 +218,13 @@ const DatePicker = ({
 
   const slots = useMemo<DatePickerSlots<DateTime>>(
     () => ({
-      actionBar: MemoizedActionBar,
+      actionBar: DateFieldActionBar,
       field: (muiProps) => renderDateField(muiProps),
-      leftArrowIcon: () => <ArrowLeftIcon />,
-      rightArrowIcon: () => <ArrowRightIcon />,
-      switchViewIcon: () => <ChevronDownIcon />,
+      leftArrowIcon: () => <commonIcons.ArrowLeftIcon />,
+      rightArrowIcon: () => <commonIcons.ArrowRightIcon />,
+      switchViewIcon: () => <commonIcons.ChevronDownIcon />,
     }),
-    [renderDateField],
+    [commonIcons, renderDateField],
   );
```

Comment:
Why are we loading the icons like this? Are we taking them in as props or something?

Reply:
They are being returned from the hook in an object called `commonIcons` Rather than importing them into every date picker variation manually

Reply:
https://github.com/okta/odyssey/pull/2391/files#diff-8f144c0d713625fb69d7e61a9e02b46ffbf3ab495fefb4ebaa883d8f2f99c58eR262